src/chebcoefs.jl

```diff
@@ -0,0 +1,43 @@
+"""
+   plan_fft(vals::AbstractArray{<:Number,N})
+
+Create an FFTW real-to-real (R2R) transformation plan for the first axis of a given multidimensional array `vals`. 
+In practise, the `vals` array is the power spectrum P(k,χ). The first axis should then contain the wavenumbers `k`.
+
+# Arguments
+- `vals::AbstractArray{<:Number, N}`: The input array of any numerical type with `N` dimensions.
+
+# Returns
+- `p::FFTW.rFFTWPlan`: An FFTW plan object for transforming `vals` with the appropriate R2R transformations. This plan can be applied using the `*` operator (e.g., `transformed_vals = p * vals`).
```

Comment:
Make more explicit the R2R. It should be clear from the context, but explicit is better than implicit.
```suggestion
- `p::FFTW.rFFTWPlan`: An FFTW plan object for transforming `vals` with the appropriate real to real transformations. This plan can be applied using the `*` operator (e.g., `transformed_vals = p * vals`).
```